keentools/utils/grace_timer.py

```diff
@@ -0,0 +1,59 @@
+# ##### BEGIN GPL LICENSE BLOCK #####
+# KeenTools for blender is a blender addon for using KeenTools in Blender.
+# Copyright (C) 2019-2022  KeenTools
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# ##### END GPL LICENSE BLOCK #####
+
+from typing import Optional
+
+from ..addon_config import Config, get_operator, ErrorType
+from .timer import KTTimer
+from ..preferences.operators import get_product_license_manager
+from ..blender_independent_packages.pykeentools_loader import module as pkt_module
+
+
+class KTGraceTimer(KTTimer):
+    def __init__(self, product: str='facebuilder'):
```

Comment:
I'd rather not use this default value because it makes it easier to forget to set a correct plugin.